activesupport/lib/active_support/deprecation/reporting.rb

```diff
@@ -37,6 +41,44 @@ def silence(&block)
         @silenced_thread.bind(true, &block)
       end
 
+      # Allow previously disallowd deprecation warnings within the block.
+      # <tt>allowed_warnings<tt> can be an array containing strings, symbols, or regular
+      # expressions. (Symbols are treated as strings). These are compared against
+      # the text of deprecation warning messages generated within the block.
+      # Matching warnings will be exempt from the rules set by
+      # +ActiveSupport::Deprecation.disallowed_warnings+
+      #
+      # The optional <tt>if:</tt> argument accepts a truthy/falsy value or an object that
+      # responds to <tt>.call</tt>. If truthy, then matching warnings will be allowed.
+      # If falsey then the method yields to the block without allowing the warning.
```

Comment:
To me, the spelling "falsy" reads a little better.

Reply:
Looking at the Rails and Ruby source, both spellings are used. I'm going to leave this as-is just to avoid a little bike-shedding but thanks for the suggestion.